client/src/outside/components/EhrSeedDataDialog.vue

```diff
@@ -127,11 +147,50 @@ export default {
       seedData.ehrData = JSON.parse(this.ehrDataString)
       this.$refs.theDialog.onClose()
       if (this.actionType === EDIT_ACTION) {
-        StoreHelper.updateSeed(this, this.seedId, seedData)
+        await StoreHelper.updateSeed(this, this.seedId, seedData)
       } else if (this.actionType === CREATE_ACTION) {
-        StoreHelper.createSeed(this, seedData)
+        const prevSeedList = StoreHelper.getSeedDataList().map(d=>JSON.stringify(d))
+        const currSeedList = await StoreHelper.createSeed(this, seedData)
+        const createdSeed = currSeedList.find(s => !prevSeedList.includes(JSON.stringify(s)))
+        this.seedId = createdSeed._id
```

Comment:
Please explain what these lines are doing with a comment

Reply:
This is done in order to compare a list of objects before and after the upload, so that, the created seed can be filtered out in order to satisfy the need of seedId when uploading the file. I have added a comment to the code and have made another PR (#580). Let me know if you have any further concerns or questions about this. Thank you!